app/src/main/res/drawable/thumb_mapbox.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="800dp"
```

Comment:
This is too big for android to render reliably, we try to keep under 240

Reply:
I reduced the width and height without changing the viewportWidth and viewportHeight. Is that enough?